_component_design/buttons.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Buttons
+title: Simple buttons
```

Comment:
Also, by changing this title, the page is now out of alphabetical order on the family page. You need to change the title of the file, as well, to avoid that problem.